dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -36,15 +36,15 @@
     %h2= level.title
   %div= render(inline: level.markdown_instructions, type: :md)
 
-  - if level.allow_user_uploads
+  - if level.allow_user_uploads == 'true'
```

Comment:
:| this seems like an easy mistake to make. is it possible to implement level.allow_user_uploads as a boolean? or implement level.allow_user_uploads?